### DISCUSSION TITLE
Negative and Positive metrics simultaneously in Stacked Bar Chart

### DISCUSSION THREAD

**Discussion by errantmind** [f5/unovis#373] [Q&A]
I have a stacked bar chart where sometimes one of the metrics (the first bar) is negative, while the other two metrics are positive. When the chart renders in these conditions, the bars are not displayed as expected, where expected would be the negative bar starting at zero and going to the left and the positive metric bars starting at zero and going to the right. 

The negative metric 'bar' sometimes doesn't start at zero and extend to the negative value, while the second metric doesn't start at zero and instead starts at the negative value of the first metric.

Is there a way to get the Stacked Bar chart to display as I might expect it to in this situation? Perhaps the stacked bar chart is not supposed to be used like this.

**Comment by rokotyan** [comment]
Hi @errantmind, can you please share your data so that we can reproduce the problem? The Stacked Bar component has two stacks: positive and negative. So all positive values will be added to the positive stack, and so on.

**Reply by errantmind** [reply]
I'll try to get an example together this week. I worked around it in the meantime.

**Reply by KABBOUCHI** [reply]
having the same issue, example:

```ts
const x = (_: any, i: number) => i;
const y = [(d: any) => d.a, (d: any) => d.b];

const data = [
  {
    a: 31733,
    b: 566959,
  },
  {
    a: 16827,
    b: -180228,
  },
];
```

https://codesandbox.io/p/devbox/wonderful-snyder-h8zpzs?file=%2Fsrc%2FApp.vue%3A27%2C36
